vertical-pod-autoscaler/pkg/updater/eviction/pods_eviction_restriction_test.go

```diff
@@ -322,11 +323,11 @@ func TestEvictReplicatedByReplicaSet(t *testing.T) {
 	}
 
 	for _, pod := range pods[:2] {
-		err := eviction.Evict(pod, test.FakeEventRecorder())
+		err := eviction.Evict(pod, getBasicVpa(), test.FakeEventRecorder())
```

Comment:
Is there any reason why not doing something like:
```
basicVpa := getBasicVpa()
for _, pod := range pods[:2] {
    err := eviction.Evict(pod, basicVpa, test.FakeEventRecorder())
    assert.Nil(t, err, "Should evict with no error")
}
for _, pod := range pods[2:] {
    err := eviction.Evict(pod, basicVpa, test.FakeEventRecorder())
    assert.Error(t, err, "Error expected")
}

Reply:
Fixed in https://github.com/kubernetes/autoscaler/pull/7413/commits/5cbf777d90ff7d21c4bc3c190fd6aaa7fade5099